packages/governance/test/unitTests/test-paramGovernance.js

```diff
@@ -209,6 +221,26 @@ test('change multiple params', async t => {
       timer,
     );
 
+  /** @type {GovernedPublicFacet<unknown>} */
+  const publicFacet = await E(governorFacets.creatorFacet).getPublicFacet();
+  const notifier = makeNotifierFromAsyncIterable(
+    await E(publicFacet).getSubscription(),
+  );
+  const update1 = await notifier.getUpdateSince();
+  // this part of the update has objects that can't be literals here for deepEqual
```

Comment:
arguments to `deepEqual` can be computed, no?

Reply:
true, so the statement is misleading. (it does have objects that can't be literals, but that's not a sufficient reason to not use deepEqual). what I was running into was the instances and handles for invitation, which I didn't know how construct, and there were so many objects that would have to be constructed for deepEqual that I didn't think it was worth it. This one isn't a test of governing, just governed values being published.

Do you think this PR should use deepEqual and compute the values? If so, any utilities for doing that the electorate invitation?

If the code is okay, I can update the comment to:
```
// constructing the fixture to deepEqual would complicate this with insufficient benefit
```